src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -200,83 +205,21 @@ public static boolean nativeCompile(Path buildRoot, ProjectConfiguration config,
         return compile;
     }
 
-    public static boolean nativeLink(Path buildRoot, ProjectConfiguration config) throws IOException, InterruptedException {
-        Objects.requireNonNull(config,  "Project configuration can't be null");
+    boolean nativeLink() throws IOException, InterruptedException {
```

Comment:
why is it not public? How are we going to call it from the plugins?

Reply:
My fault. Will be fixed shortly.